### DISCUSSION TITLE
Best Practise model continual training

### DISCUSSION THREAD

**Discussion by lschneidpro** [mlflow/mlflow#7383] [Q&A]
Hi there,
I was wondering how to use MLflow when re-training model over time in production. I agree that I can manage the model versions through the Model Registry interface. Regarding Experiment tracking, should I perform the different re-training in different experiments or keep all in the same experiment. Thanks for your answer

**Comment by cao-nv** [comment]
Since MLFlow does not require versions of a model to be produced from runs of the same experiment, you can choose the option you are comfortable with.
